source/features/useful-not-found-page.tsx

```diff
@@ -145,13 +145,16 @@ async function showAlternateLink(): Promise<void> {
 
 function init(): false | void {
 	const parts = parseCurrentURL();
-	if (parts.length <= 1 || !select.exists('[alt*="This is not the web page you are looking for"]')) {
```

Comment:
If I remember correctly we needed this to detect if it was a 404

Reply:
Yes
https://github.com/sindresorhus/refined-github/commit/bfdad4b4ea240c93ab0c51ff4e2a71a5b3849c94
https://github.com/sindresorhus/refined-github/issues/1856

Reply:
I tried it out with the new code, and it does not trigger the `useful-not-found-page` on the advanced search page. I think it's because the page does not count as a 404 page anymore (`pageDetect.is404`).

Reply:
> If I remember correctly we needed this to detect if it was a 404

Yeah, and that seems wrong. We already have a `is404` that should that. If it doesn't work, _that_ should be fixed. But by the looks of it, `is404` already does not include the linked page.

Reply:
https://github.com/sindresorhus/refined-github/issues/4008#issuecomment-894979392